### ISSUE TITLE
Android drop down menu items alignment

### ISSUE THREAD

**Issue by M3Rocket** [dasilva333/TowerGhostForDestiny#119]
Since the Menu button on the top right of the screen, I suggest instead of making the menu items left justified, to move all the menu items to be aligned to be right justified. As is, it just looks out of place with a lot of wasted space--especially in Landscape mode.

![screenshot_2015-04-30-17-45-53](https://cloud.githubusercontent.com/assets/3769123/7425219/b4d51c40-ef61-11e4-84c4-99c7fc33bc04.png)

**Comment by pattont** [comment]
Agreed, it's weird right now

**Comment by Doubleken** [comment]
Or put all menu on top like Computer version, can swap right or left to display?

**Comment by dasilva333** [comment]
This will look better after some polish I'm sure of it

**Comment by M3Rocket** [comment]
Yep. If there's a way to turn it into a slideout/flyout menu (left or right doesn't matter), that would be ideal I think.
